filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -899,21 +902,43 @@ void OpenGLDriver::framebufferTexture(backend::TargetBufferInfo const& binfo,
         }
     } else
 #endif
-    { // here we emulate ext.EXT_multisampled_render_to_texture
+    if (!any(t->usage & TextureUsage::SAMPLEABLE)) {
```

Comment:
I think this test is not enough. We also need a sidecar if the target render buffer is not multi-sampled.
That is, if the user is asking for a "magic resolve" into a render buffer.

Reply:
If the target render buffer is not multi-sampled, then this code is unreachable; see line 850.

Reply:
No no that's not what I mean. I meant if the render buffer is not multi-sampled, but the render target is. Again, this is the case of a "magic resolve" into a render buffer.

Reply:
This is complicated!  Okay I added another commit to make this change, and thankfully WebGL is still working.